src/coreclr/jit/emitpub.h

```diff
@@ -76,7 +76,7 @@ const char* emitOffsetToLabel(unsigned offs);
 /*                   Output target-independent instructions             */
 /************************************************************************/
 
-void emitIns_J(instruction ins, BasicBlock* dst, int instrCount = 0);
+void emitIns_J(instruction ins, BasicBlock* dst, int instrCount = 0, bool isJmpAlways = false);
```

Comment:
It looks like `isJmpAlways` is only implemented for x64. In that case, I think we need to move this declaration to the arch-specific emit*.h files, and only add the argument for x64.

Reply:
Done for both this method and the caller which needed the extra argument as well.